### PR TITLE
New tokens: Alpha alpha

### PR DESCRIPTION
**Pull request by therealMikkoK**

### CODE REVIEW

**Comment by freeatnet** [comment]
@therealMikkoK BNK removed.

File: tokens/0xfedae5642668f8636a11987ff386bfd215f942ee.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+__FORKDELTA_CUSTOM_SYMBOL: AVOID CONFLICT WITH 0x562952c749d05dca4cd004489a153c7ee7e58240
```
Comment (freeatnet) [inline tokens/0xfedae5642668f8636a11987ff386bfd215f942ee.yaml]: It appears as though [PallyCoin PAL by pally.co](https://github.com/forkdelta/tokenbase/blob/master/tokens/0x562952c749d05dca4cd004489a153c7ee7e58240.yaml) is already assigned PAL symbol. We need to figure out a workable arrangement with policypal.network and pally.co folks to resolve this conflict. The custom assigned ticker may go into the `__FORKDELTA_CUSTOM_SYMBOL` field here.

Comment (therealMikkoK) [reply tokens/0xfedae5642668f8636a11987ff386bfd215f942ee.yaml]: OK. does this mean we wait for this, before the merge?

Comment (freeatnet) [reply tokens/0xfedae5642668f8636a11987ff386bfd215f942ee.yaml]: @therealmikkok well, we need to assign it a unique symbol/ticker. We can either do it on our own, or offer the parties (and policypal.network peeps specially) a chance to pick their own. Can we reach out to them?

**Comment by capital-technologies** [comment]
@therealMikkoK @freeatnet Can you add our assets in the next batch or even this? We have closed the crowdsale and now no new tokens can be minted. Or issue id is **[#1629](https://github.com/forkdelta/tokenbase/issues/1629)** and our PR is **[#1628](https://github.com/forkdelta/tokenbase/pull/1628)** Thank you!
Best regards!

**Comment by freeatnet** [comment]
@therealMikkoK I have pulled PAL PolicyPal into [a separate branch](https://github.com/forkdelta/tokenbase/compare/new-tokens/1075-pal-policypal?expand=1).

**Comment by therealMikkoK** [comment]
ready to merge, will continue with PAL on the next batch.

**Pull request merged**